### PR TITLE
Added trailmaking task to the active tasks and result archive.

### PR DESCRIPTION
**Pull request by syoung-smallwisdom**
This is an example for the steps required to add a task (that has been defined in ResearchKit) to the BridgeAppSDK. This includes:

1. Update the submodule to point to Sage's current default branch of https://github.com/Sage-bionetworks/ResearchKit with your changes built in. (Reference pull request to that branch.

Sage PR: https://github.com/Sage-Bionetworks/ResearchKit/pull/40
ResearchKit PR: https://github.com/ResearchKit/ResearchKit/pull/880

2. Add a `taskType` to the `SBAActiveTask` including a method for creating the task.
3. Add an override to `ORKResults+ResultsExtension` to convert the ORKResult for the task to a schema that Bridge can translate into a Synapse Table.
4. Add a unit test for the task to `SBAActiveTaskTests`
5. Add the task to the sample app BridgeInfo.plist

From within the [Bridge Researcher UI](https://research.sagebridge.org), you will need to add the following:

1. Task Identifier - This maps to the `taskMapping` in the BridgeInfo.plist and is used to map to the `SBBTaskReference` included with a scheduled activity.

screenshot 2016-12-15 10 27 49

2. Schema - This maps to the `schemaMapping` in the BridgeInfo.plist and the schema you have defined in your override within `ORKResult+ResultExtensions`. Note: the "trailmaking.json" suffix defined for this example is determined as "<result.identifier>.json"

screenshot 2016-12-15 10 15 17

3. Schedule - Using the scheduler, you can then schedule the tasks.

screenshot 2016-12-15 10 27 32

### CODE REVIEW

File: BridgeAppSDK/SBAActiveTask.swift [hunk 1]
```diff
@@ -44,6 +44,7 @@ public enum SBAActiveTaskType {
     case walking
     case tremor
     case moodSurvey
+    case trailmaking
```
Comment (Erin-Mounts) [inline BridgeAppSDK/SBAActiveTask.swift:47]: Would it be useful instead to use this an example of adding a custom task at the app level? Or is it more useful to have it built in to the SDK do you think?

**Review by Erin-Mounts — APPROVED** [review]
One question, otherwise looks good, merge when ready.

File: BridgeAppSDK/SBAActiveTask.swift (hunk 1 above)
Comment (syoung-smallwisdom) [reply BridgeAppSDK/SBAActiveTask.swift:47]: I was figuring on including it here and sending Faraz a link to this PR so he could see what changes are required to implement, though that does remind me to add it to the sample app as well.

**Review by Erin-Mounts — APPROVED** [review]
Looks good.

**Pull request merged**